src/renderer/screens/dashboard/index.js

```diff
@@ -81,7 +84,9 @@ const DashboardPage = ({ saveSettings }: Props) => {
         totalCurrencies={totalCurrencies}
       />
       <Box flow={7}>
-        {totalAccounts > 0 ? (
+        {!hasInstalledApps ? (
+          <EmptyStateInstalledApps />
+        ) : totalAccounts > 0 ? (
```

Comment:
mmh to me this should be the first condition instead:

if user have totalAccounts, we really should display the normal portfolio.

it's only if user have no accounts that we then start displaying the relevant placeholder.

Reply:
will change or not once we all agree on a flow